source/management_api/resource/streamsResource.js

```diff
@@ -12,12 +12,24 @@ var logger = require('./../logger').logger;
 // Logger
 var log = logger.getLogger('StreamsResource');
 
+const convertV1Stream = function (stream) {
```

Comment:
Conversion should not be needed if the data is already in v11 format, according to section 5.3 in https://github.com/open-webrtc-toolkit/owt-server/blob/master/doc/servermd/RESTAPI.md